packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -566,14 +586,25 @@ export class Function extends FunctionBase {
     if (props.securityGroup && props.allowAllOutbound !== undefined) {
       throw new Error(`Configure 'allowAllOutbound' directly on the supplied SecurityGroup.`);
     }
-
-    const securityGroup = props.securityGroup || new ec2.SecurityGroup(this, 'SecurityGroup', {
-      vpc: props.vpc,
-      description: 'Automatic security group for Lambda Function ' + this.node.uniqueId,
-      allowAllOutbound: props.allowAllOutbound
-    });
-
-    this._connections = new ec2.Connections({ securityGroups: [securityGroup] });
+	
+    let securityGroups: ec2.ISecurityGroup[];
+    
+    if (props.securityGroup && props.securityGroups) {
+      throw new Error('Only one of the function props, securityGroup or securityGroups, is allowed');
+    }
+    
+    if (props.securityGroups) {
+      securityGroups = props.securityGroups;
+    } else {
+      const securityGroup = props.securityGroup || new ec2.SecurityGroup(this, 'SecurityGroup', {
+        vpc: props.vpc,
+        description: 'Automatic security group for Lambda Function ' + this.node.uniqueId,
+        allowAllOutbound: props.allowAllOutbound
+      });
+      securityGroups = [securityGroup];
+    }
+    
+    this._connections = new ec2.Connections({ securityGroups: securityGroups });
```

Comment:
```suggestion
    this._connections = new ec2.Connections({ securityGroups });
```

`@aws-cdk/aws-lambda: ERROR: /codebuild/output/src927959865/src/github.com/aws/aws-cdk/packages/@aws-cdk/aws-lambda/lib/function.ts:607:47 - Expected property shorthand in object literal ('{securityGroups}').`

Reply:
Fixed.